packages/wallets/src/features/accounts/screens/ManualDocumentUpload/components/PassportDocumentUpload/PassportDocumentUpload.tsx

```diff
@@ -1,18 +1,62 @@
 import React from 'react';
-import { WalletButton, WalletButtonGroup } from '../../../../../../components/Base';
+import { Dropzone } from '../../../../../../components';
+import { Divider, WalletText, WalletTextField } from '../../../../../../components/Base';
+import useDevice from '../../../../../../hooks/useDevice';
+import ClearPhoto from '../../../../../../public/images/accounts/clear-photo.svg';
+import ClockIcon from '../../../../../../public/images/accounts/clock-icon.svg';
+import ImageIcon from '../../../../../../public/images/accounts/image-icon.svg';
+import LessThanEightIcon from '../../../../../../public/images/accounts/less-than-eight-icon.svg';
+import PassportPlaceholder from '../../../../../../public/images/accounts/passport-placeholder.svg';
+import Calendar from '../../../../../../public/images/calendar.svg';
+import { DocumentRuleHint } from '../DocumentRuleHint';
+import './PassportDocumentUpload.scss';
 
-type TProps = {
-    setSelectedDocument: (document: string) => void;
-};
+const documentRules = [
+    {
+        description: 'A clear colour photo or scanned image',
+        icon: <ClearPhoto />,
+    },
+    {
+        description: 'JPEG, JPG, PNG, PDF, or GIF',
+        icon: <ImageIcon />,
+    },
+    {
+        description: 'Less than 8MB',
+        icon: <LessThanEightIcon />,
+    },
+    {
+        description: 'Must be valid for at least 6 months',
+        icon: <ClockIcon />,
+    },
+];
+
+const PassportDocumentUpload = () => {
+    const { isDesktop } = useDevice();
 
-const PassportDocumentUpload: React.FC<TProps> = ({ setSelectedDocument }) => {
     return (
-        <div data-testid='dt_passport-document-upload'>
-            <div>
-                <WalletButtonGroup>
-                    <WalletButton onClick={() => setSelectedDocument('')} text='Back' variant='outlined' />
-                    <WalletButton onClick={() => setSelectedDocument('')} text='Next' />
-                </WalletButtonGroup>
+        <div className='wallets-passport-document-upload' data-testid='dt_passport-document-upload'>
+            <WalletText>First, enter your Passport number and the expiry date.</WalletText>
+            <div className='wallets-passport-document-upload__input-group'>
+                <WalletTextField label='Passport number*' />
+                <WalletTextField label='Expiry date*' renderRightIcon={() => <Calendar />} type='date' />
+            </div>
+            <Divider />
+            <div className='wallets-passport-document-upload__document-section'>
+                <WalletText>Next, upload the page of your passport that contains your photo.</WalletText>
+                <Dropzone
+                    buttonText='Drop file or click here to upload'
+                    description='Upload the page of your passport that contains your photo.'
```

Comment:
Translation?

Reply:
We don't have any translation in the wallets yet, will decide on this after first release.